common/hashing_test.cpp

```diff
@@ -161,43 +196,51 @@ TEST(HashingTest, PairsAndTuples) {
   EXPECT_THAT(HashValue(std::tuple(1, 0)), Eq(hash_10));
   EXPECT_THAT(HashValue(std::tuple(1, 1)), Eq(hash_11));
 
-  // Integers in tuples should work the same as outside of tuples w.r.t.
-  // converting between different integer types.
-  for (int i : {0, 1, 2, 3, 42}) {
+  // Integers in tuples should work the same as outside of tuples.
```

Comment:
Could you be more specific about what about integers in vs. out of tuples is working the same?